src/blazesym_int.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#ifdef USE_BLAZESYM
+#include <blazesym.h>
+
+#define BLAZESYM_OPT                                                           \
+  _Pragma("GCC diagnostic ignored \"-Wmissing-field-initializers\"")
```

Comment:
Use pragma push and pragma pop instead? This seems unbounded - a transitive include of this file could add this for a bunch of TUs. Would be good to tightly scope where this warning is disabled so it's obvious what it's there for.